src/PrestaShopBundle/Form/Admin/Configure/ShopParameters/ProductPreferences/PageType.php

```diff
@@ -37,27 +37,84 @@
  * Class generates "Product page" form
  * in "Configure > Shop Parameters > Product Settings" page.
  */
-class PageType extends AbstractType
+class PageType extends TranslatorAwareType
 {
     /**
      * {@inheritdoc}
      */
     public function buildForm(FormBuilderInterface $builder, array $options)
     {
         $builder
-            ->add('display_quantities', SwitchType::class)
-            ->add('display_last_quantities', IntegerType::class)
-            ->add('display_unavailable_attributes', SwitchType::class)
-            ->add('allow_add_variant_to_cart_from_listing', SwitchType::class)
+            ->add('display_quantities', SwitchType::class, [
+                'label' => $this->trans(
+                    'Display available quantities on the product page',
+                    'Admin.Shopparameters.Feature'
+                ),
+            ])
+            ->add('display_last_quantities', IntegerType::class, [
+                'label' => $this->trans(
+                    'Display remaining quantities when the quantity is lower than',
```

Comment:
I tested that feature but couldn't make it work. Where are the remaining quantities supposed to appear? 🤔

Reply:
It impacts the configuration `PS_LAST_QTIES`. That impacts in front the label `Last items in stock`.